packages/office-ui-fabric-react/src/components/Persona/Persona.test.tsx

```diff
@@ -22,10 +22,15 @@ const STYLES = {
  * function to override the default onRender callbacks
  */
 export const wrapPersona = (
-  example: IPersonaSharedProps
+  example: IPersonaSharedProps,
+  isCoin: boolean = false
 ): ((coinProps: IPersonaProps, defaultRenderer: IRenderFunction<IPersonaProps>) => JSX.Element | null) => {
   return (coinProps, defaultCoinRenderer): JSX.Element | null => {
-    return defaultCoinRenderer(coinProps);
+    return isCoin ? (
```

Comment:
From the name, it is unclear to me what "isCoin" means in this case. Would prefer a name that explains what the result of setting the parameter would be